src/content/cloud/personal-access-tokens.mdx

```diff
@@ -24,6 +24,16 @@ Personal Access Tokens are useful when scripting, or when configuring [Preview E
 
 > Warning: Treat your tokens like passwords and keep them secret. Always use tokens as environment variables instead of hardcoding them into your programs.
 
+### Token Expiration
+
+> A token is valid for 180 days. Once it expires, you will need to delete it and create a new one.
+
+Once a token has expired or is about to expire (in less than a week), a banner will be presented alerting you to the fact, so you can take the appropriate measures, like rotating it.
```

Comment:
```suggestion
A banner will be displayed a week before your token expires to remind you of this.  We recommend that you take the appropriate measures once you see this banner in order to prevent any potential disruptions.
```